plugins/linux-capture/xcompcap-main.cpp

```diff
@@ -395,16 +409,25 @@ void XCompcapMain::updateSettings(obs_data_t *settings)
 		glBindTexture(GL_TEXTURE_2D, 0);
 	}
 
-	const int attrs[] =
+	const int attrs_alpha[] =
 	{
 		GLX_BIND_TO_TEXTURE_RGBA_EXT, GL_TRUE,
 		GLX_DRAWABLE_TYPE, GLX_PIXMAP_BIT,
 		GLX_BIND_TO_TEXTURE_TARGETS_EXT, GLX_TEXTURE_2D_BIT_EXT,
 		GLX_ALPHA_SIZE, 8,
-		GLX_DOUBLEBUFFER, GL_FALSE,
```

Comment:
I notice that `GLX_DOUBLEBUFFER, GL_FALSE,` seem to be completely removed.  Did anything rely on them?  Was their functionality here replaced elsewhere that I'm not seeing?

Reply:
Previously, the capture worked by creating an "ideal" GLXFBConfig for capture, but that didn't work for every window. Now, under the model of attempting to inherit the GLXFBConfig from the window, by having that condition meant that inheriting from double-buffered windows was impossible.